src/submitWork.js

```diff
@@ -2,6 +2,6 @@ const {
   submitWork
 } = require('./messageContants');
 
-module.exports = (options) => {
-  options.client.write(submitWork.replace("<worker.name>", options.worker_name).replace("<jobID>", options.job_id).replace("<ExtraNonce2>", options.extranonce2).replace("<ntime>", options.ntime).replace("<nonce>", options.nonce));
+module.exports = (worker, job_id, extranonce2, ntime, nonce, client) => {
```

Comment:
Let's keep this as `options` because it is easier to pass parameters without remembering the sequence. In future when I move the code to typescript, it would be better to have more types.